frontend/packages/console-shared/src/components/formik-fields/SyncedEditorField.tsx

```diff
@@ -12,6 +12,7 @@ type EditorContext = {
   name: string;
   editor: React.ReactNode;
   isDisabled?: boolean;
+  sanitizeTo?: (preFormData: any) => any;
```

Comment:
@rohitkrai03 I added this - let me know if you have any concerns about this functionality. I needed a way to get my root data back to a stable state when going from YAML => Form.

Reply:
Yeah, no problem. This looks good to me.